docs/sources/configuration/_index.md

```diff
@@ -1218,6 +1222,11 @@ lifecycler:
   # CLI flag: -ingester.lifecycler.interface
   [interface_names: <list of strings> | default = [<private network interfaces>]]
 
+  # Enable IPv6 support. Required to make use of IP addresses from IPv6
+  # interfaces.
+  # CLI flag: -ingester.enable-inet6
+  [enable_inet6: <boolean> | default = false]
```

Comment:
Is it possible to rename this to `enable_ipv6`?

Reply:
Would mean to fix in main too, but sure we can do.

Reply:
Can we not alias this field differently in the struct that embeds it?